content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -289,6 +289,75 @@ The Pod which is already running correctly would be killed by `activeDeadlineSec
 The name of each app and init container in a Pod must be unique; a
 validation error is thrown for any container sharing a name with another.
 
+### Sidecar containers and restartPolicy
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Thanks to a newly available feature gate named `SidecarContainers`, each
+init container can have its own `restartPolicy`, independent of the Pod and the
+other init containers, as well as any regular [probe](/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe).
+
+If an init container is created with a `restartPolicy` set to `Always`, it will
+start and remain running during the entire life of the Pod, which is useful for
+running supporting services isolated from the main application container.
+You can also specify a `startupProbe` and/or a `readinessProbe` to help kubelet
+determine the `ready` state of these containers.
+
+Since these containers are defined as init containers, they benefit from the same
+ordering and sequential guarantees as other init containers, allowing them to
+be mixed with other init containers into complex Pod initialization flows.
+
+The only difference is that they are not required to complete before the next
+init container starts, which is impossible, so next init container will start
+after the current container has been marked as `ready`.
```

Comment:
https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/753-sidecar-containers/README.md#pod-startup-completed-condition

marked as `started`. ?

Reply:
Hmm right, but then what's the purpose of the readinessProbe in that case?

Reply:
It determines the ready state of the restartable init container, and the pod's ready state should include not only regular containers but also restartable init containers.

e.g. If the proxy sidecar is not ready, the pod should not get any traffic from the service.

Reply:
Can you suggest a modification or change the PR commit? I am in Paris without a computer until Friday, and I think doc freeze is today?

Reply:
> done

I guess docs can be reviewed and merged until the 8th of August?